linera-execution/src/evm/revm.rs

```diff
@@ -274,55 +280,87 @@ fn address_to_user_application_id(address: Address) -> ApplicationId {
 
 /// Some functionalities from the BaseRuntime
 #[derive(Debug, Serialize, Deserialize)]
-enum BasePrecompileTag {
+enum BaseRuntimePrecompile {
     /// Key prefix for `chain_id`
     ChainId,
     /// Key prefix for `application_creator_chain_id`
     ApplicationCreatorChainId,
     /// Key prefix for `chain_ownership`
     ChainOwnership,
     /// Key prefix for `read_data_blob`
-    ReadDataBlob,
+    ReadDataBlob { hash: CryptoHash },
```

Comment:
This isn't a prefix anymore, is it? same for `AssertDataBlobExists`

Reply:
Yes, thank you find finding the oversight.